tictactoe/test/utilities_tests.erl

```diff
@@ -0,0 +1,140 @@
+-module(utilities_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+move_test() ->
+  InvalidMove = utilities:validate_move_position(console_io:validate_move_format({ok,["1"]}), [x,2,3,4,5,6,7,8,9]),
+  ValidMove = utilities:validate_move_position(console_io:validate_move_format({ok,["1"]}), [1,2,3,4,5,6,7,8,9]),
+  [{  "checks validity and marks board when move is valid, otherwise returns invalid info tuple",
+      ?assertEqual({invalid,[1,2,3,4,5,6,7,8,x],o}, 
+                   utilities:move(InvalidMove,[1,2,3,4,5,6,7,8,x],o)),
+      ?assertEqual([x,2,3,4,5,6,7,8,9], 
+                   utilities:move(ValidMove,[1,2,3,4,5,6,7,8,9],x))}].
+
+mark_board_test() ->
+  [{  "returns board with marker placed",
+      ?assertEqual(
+        ["X",2,3,4,5,6,7,8,9],
+        utilities:mark_board(1, [1,2,3,4,5,6,7,8,9], "X")),
+      ?assertEqual(
+        ["O",2,3,4,5,6,7,8,9],
+        utilities:mark_board(1, [1,2,3,4,5,6,7,8,9], "O"))}].
+
+change_player_test() ->
+  [{  "changes player marker",
+      ?assertEqual(o,utilities:change_player(x)),
+      ?assertEqual(x,utilities:change_player(o))}].
+
+whos_in_play_test() -> 
+  [{  "using only the board; determines which player is currently at play (their turn)",
```

Comment:
Should have a couple more test conditions here to verify this method works beyond each player's first turn.